Doc/library/exceptions.rst

```diff
@@ -855,6 +855,10 @@ The following exceptions are used as warning categories; see the
 
    Base class for warnings related to :class:`bytes` and :class:`bytearray`.
 
+.. exception:: Py2xWarning
+
+   Base class for warnings related to 3.x compatibility.
```

Comment:
Should it be "related to 2.x compatibility"?

Reply:
Seems that we are warning when one write 2.x code, to signal for 3.x compatibility.

Reply:
So it's correct? If so, OK :)